src/NuGet.Licenses/NuGet.Licenses.csproj

```diff
@@ -130,6 +130,9 @@
     <PackageReference Include="Microsoft.Extensions.Logging">
       <Version>2.2.0</Version>
     </PackageReference>
+    <PackageReference Include="Newtonsoft.Json">
```

Comment:
do we need `Newtonsoft.json` package reference for compile time here?

Reply:
Without direct reference client will resolve to some lower version of newtonsoft.json, that satisfies all direct dependencies, so if we want to force 13.0.1 we have to have it here.